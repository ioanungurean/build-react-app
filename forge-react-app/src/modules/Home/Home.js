import React, { Component } from 'react';

import './Home.scss';

class Home extends Component {
  render() {
    return (
      <div className="home">
        Home
      </div>
    );
  }
}

Home.displayName = 'Home';
export default Home;
