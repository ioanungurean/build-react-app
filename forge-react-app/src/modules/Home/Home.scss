@import 'xbem.scss';
@import 'layout.scss';
@import 'patterns.scss';

@include block(home) {}
