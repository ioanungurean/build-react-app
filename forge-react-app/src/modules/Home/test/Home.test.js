import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

describe('<Home />', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<Home />);

    expect(wrapper).toMatchSnapshot();
  });
});
