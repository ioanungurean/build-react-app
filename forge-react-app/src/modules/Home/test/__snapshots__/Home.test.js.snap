// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render without throwing an error 1`] = `
<div
  className="home"
>
  Home
</div>
`;
